### ISSUE TITLE
Boxplot backwards incompatibility 

### ISSUE THREAD

**Issue by TGabor** [IoSR-Surrey/MatlabToolbox#2]
Hi,

first, thanks for your great toolbox. 
I just tried the new version of boxPlot (commit 4588ab25a4f391a0fa9f98e2c0d77dbb7da4f180)
and noticed that it produces different resuts for the same input data compared to the old box_plot.m I have from FEX:
% Last changed:     $Date: 2015-12-07 10:55:15 +0000 (Mon, 07 Dec 2015) $
% Last committed:   $Revision: 451 $)

For example, the quantiles have a different extent, although there are no outliers.
I generated a gif that illustrates the problem (see e.g. the 16384 case in the gif)
![output](https://cloud.githubusercontent.com/assets/1191021/16688046/61e3d69a-451c-11e6-93b4-215892849270.gif)

Is there a change in the outlier computation as well? The old box_plot produces a different set of outliers than the new one (see e.g. the blue box for the 512 case in the gif)
![output2](https://cloud.githubusercontent.com/assets/1191021/16688150/e6ab7a9a-451c-11e6-98d7-16a10ca2637c.gif)

I use Matlab2015b under Win7.

I uploaded a minimum working example  (test_boxplot.m) along with the data here:
[debug_boxPlot.zip](https://github.com/IoSR-Surrey/MatlabToolbox/files/354290/debug_boxPlot.zip)

**Comment by chummersone** [comment]
Many thanks for the bug report, and for going to the effort of making an MWE. It looks like the differences are due to an update I made to the quantile function; I [added a weighting option](https://github.com/IoSR-Surrey/MatlabToolbox/commit/96f73a93c628be86f53bdbbf75bcd89121ccb924). It shouldn't give different numbers when no weighting is specified, but it obviously does (though I'm sure it didn't when I tested it). I'll have to investigate...

**Comment by chummersone** [comment]
Hi. Thanks so much for reporting the bug, and producing the MWE. The different results are as a result of the default 'method' parameter, which was erroneously specified as 'R-5' in the newer version (despite saying 'R-8' in the help), and specified as 'R-8' in the older version. I've corrected the error, and uploaded a fix to the repo. Note that the x-displacement of the outliers may still differ because the offset is pseudo-random, but also because [I tweaked the displacement algorithm](https://github.com/chummersone/boxPlot/commit/b7664a64832f665890d34a138b475b9dbf347978).
